src/components/ModalInvoke/ModalInvoke.tsx

```diff
@@ -71,12 +71,16 @@ const ModalInvoke: React.FunctionComponent<ModalInvokeProps> = props => {
         }
     }
 
+    const okLabel = props.confirmOperation.okText || t('Ok')
```

Comment:
Optional chaining probably required for `confirmOperation` (based on original line 78)

Reply:
Optional chaining is added